website/src/examples/xhrupload/app.scss

```diff
@@ -0,0 +1,40 @@
+.uploaded-files{
+  background: rgba(240, 238, 238, 0.35);
+  padding: 10px;
+  margin-top: 10px;
+  border-radius: 5px;
+  h5 {
+    margin: 0;
+    font-size: 15px;
+    color: rgb(114, 114, 114);
+    font-weight: 500;
+  }
+  ol {
+    color: rgb(55, 55, 55);
+    margin: 0;
+    padding-left: 18px;
+    li {
+      font-size: 15px;
+      padding-top: 10px;
+      word-break: break-all;
+      a {
+        color: rgb(34, 117, 215);
+        &:hover {
+          color: rgb(34, 117, 215);
+          border-bottom: 1px solid rgb(34, 117, 215);
+        }
+      }
+    }
+  }
+}
+
+// Prettier progress bar
+.uppy-ProgressBar-inner {
+  box-shadow: none !important;
```

Comment:
I wonder if we should change the progress bar to only add the shadow when `fixed: true`?

Reply:
I thought of removing the box-shadow altogether, I think it would look better without the shadow everywhere.

Reply:
Well, there might be a case when you put it on top of a website, and then scroll to different places, over a full-width video, for example, the progress bar might be less visible, or not at all, without the shadow.